pkg/cmd/init/cmd.go

```diff
@@ -46,5 +48,9 @@ func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, stream
 	cmd.Flags().StringVar(&o.outputFile, "output-file", "", "The generated resources will be copied in the specified file")
 	cmd.Flags().BoolVar(&o.useBootstrapToken, "use-bootstrap-token", false, "If set then the boostrap token will used instead of a service account token")
 	cmd.Flags().BoolVar(&o.force, "force", false, "If set then the hub will be reinitialized")
+	cmd.Flags().StringVar(&o.registry, "image-registry", "quay.io/open-cluster-management",
+		"The name of the image registry serving OCM images.")
+	cmd.Flags().StringVar(&o.version, "version", "latest",
```

Comment:
I would call this one tag instead as version.

Reply:
renamed to tag